lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -12,11 +12,21 @@ class Results
         # ab123cd4567.xml.0001.result.xml
         RESULT_FILE_PATTERN = /(?<druid>\w+)\.xml(?:\.(?<run_index>\d+))?\.result\.xml/
 
-        def initialize(result_path:)
```

Comment:
This new method is used to use a druid to locate the latest job result file and instantiate a `Results` object for it. It's a bit clunky that the logger needs to be passed in, but that seems to be what is done elsewhere?

Reply:
Yeah, not sure what to do about passing this logger everywhere.  A problem for another day.